### ISSUE TITLE
Cannot build for Android 14

### ISSUE THREAD

**Issue by rlee1990** [GetStream/stream-video-flutter#555]
Cannot build for Android 14(34) tested with the latest Flutter stable version and stream_video_flutter version 0.3.0.
I have checked building a new project with just this package and it fails to launch. Removing the package from my app or a test app launches.
Trying to run the demo app also fails to launch.

**Comment by rlee1990** [comment]
```
E/AndroidRuntime(20116): java.lang.SecurityException: com.socialjawn.supabase.test.supabase_social_jawn_test: One of RECEIVER_EXPORTED or RECEIVER_NOT_EXPORTED should be specified when a receiver isn't being registered exclusively for system broadcasts
E/AndroidRuntime(20116): 	at android.os.Parcel.createExceptionOrNull(Parcel.java:3057)
E/AndroidRuntime(20116): 	at android.os.Parcel.createException(Parcel.java:3041)
E/AndroidRuntime(20116): 	at android.os.Parcel.readException(Parcel.java:3024)
E/AndroidRuntime(20116): 	at android.os.Parcel.readException(Parcel.java:2966)
E/AndroidRuntime(20116): 	at android.app.IActivityManager$Stub$Proxy.registerReceiverWithFeature(IActivityManager.java:5684)
E/AndroidRuntime(20116): 	at android.app.ContextImpl.registerReceiverInternal(ContextImpl.java:1852)
E/AndroidRuntime(20116): 	at android.app.ContextImpl.registerReceiver(ContextImpl.java:1792)
E/AndroidRuntime(20116): 	at android.app.ContextImpl.registerReceiver(ContextImpl.java:1780)
E/AndroidRuntime(20116): 	at android.content.ContextWrapper.registerReceiver(ContextWrapper.java:755)
E/AndroidRuntime(20116): 	at io.getstream.video.flutter.stream_video_flutter.service.utils.AndroidUtilsKt$registerReceiverAsFlow$1.invokeSuspend(AndroidUtils.kt:36)
E/AndroidRuntime(20116): 	at io.getstream.video.flutter.stream_video_flutter.service.utils.AndroidUtilsKt$registerReceiverAsFlow$1.invoke(Unknown Source:8)
E/AndroidRuntime(20116): 	at io.getstream.video.flutter.stream_video_flutter.service.utils.AndroidUtilsKt$registerReceiverAsFlow$1.invoke(Unknown Source:4)
E/AndroidRuntime(20116): 	at kotlinx.coroutines.flow.ChannelFlowBuilder.collectTo$suspendImpl(Builders.kt:322)
E/AndroidRuntime(20116): 	at kotlinx.coroutines.flow.ChannelFlowBuilder.collectTo(Unknown Source:0)
E/AndroidRuntime(20116): 	at kotlinx.coroutines.flow.CallbackFlowBuilder.collectTo(Builders.kt:336)
E/AndroidRuntime(20116): 	at kotlinx.coroutines.flow.internal.ChannelFlow$collectToFun$1.invokeSuspend(ChannelFlow.kt:60)
E/AndroidRuntime(20116): 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
E/AndroidRuntime(20116): 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
E/AndroidRuntime(20116): 	at android.os.Handler.handleCallback(Handler.java:958)
E/AndroidRuntime(20116): 	at android.os.Handler.dispatchMessage(Handler.java:99)
E/AndroidRuntime(20116): 	at android.os.Looper.loopOnce(Looper.java:205)
E/AndroidRuntime(20116): 	at android.os.Looper.loop(Looper.java:294)
E/AndroidRuntime(20116): 	at android.app.ActivityThread.main(ActivityThread.java:8177)
E/AndroidRuntime(20116): 	at java.lang.reflect.Method.invoke(Native Method)
E/AndroidRuntime(20116): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
E/AndroidRuntime(20116): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:971)
E/AndroidRuntime(20116): 	Suppressed: kotlinx.coroutines.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@ce71579, Dispatchers.Main]
E/AndroidRuntime(20116): Caused by: android.os.RemoteException: Remote stack trace:
E/AndroidRuntime(20116): 	at com.android.server.am.ActivityManagerService.registerReceiverWithFeature(ActivityManagerService.java:13927)
E/AndroidRuntime(20116): 	at android.app.IActivityManager$Stub.onTransact(IActivityManager.java:2570)
E/AndroidRuntime(20116): 	at com.android.server.am.ActivityManagerService.onTransact(ActivityManagerService.java:2720)
E/AndroidRuntime(20116): 	at android.os.Binder.execTransactInternal(Binder.java:1339)
E/AndroidRuntime(20116): 	at android.os.Binder.execTransact(Binder.java:1275)
```

**Comment by sdegenaar** [comment]
I can confirm I am experiencing the same issue

**Comment by Brazol** [comment]
Hey @rlee1990, this issue should be resolved now in version 0.3.1
